deepmd/dpmodel/model/make_base_atomic_model.py

```diff
@@ -84,6 +84,32 @@ def serialize(self) -> dict:
         def deserialize(cls):
             pass
 
+        def compute_or_load_stat(
```

Comment:
is it supposed to be an interface? if so make it an abstract method

Reply:
Maybe `BaseAtomicModel` is a better choice to have `compute_or_load_stat` as the interface, since both`DPAtomicModel` and `PairTabModel` inherit from it. But it's maybe not an abstract method, because numpy doesn't need this method. So maybe here raise NotImplementedError is better.

Reply:
The problem is that all backends are supposed to use the stat in exactly the same way, thus all the backends are assumed to provide the same interface. I support design it as abstract method of  some base class, but may not be `BaseAtomicModel`, because it does not necessarily have a descriptor, thus the statistics is different from those needs descriptor statistics. 

It should be abstract method, because the stat interfaces will be used in the same way cross descriptors and fittings. numpy may pass null stats (mean 0 std 1).

Reply:
`compute_or_load_stat` method has been changed into `BaseModel` now, which is a base class of pytorch model, but not as an abstract method.
The inheritance relationship between those class is as following:
DPAtomicModel(BaseModel, BaseAtomicModel)

Reply:
`compute_or_load_stat` method has been changed in to `BaseModel`, which is a base class of pytorch model, but as a method with `NotImplement` error and not as an abstract method. 

The inheritance relationship of these classed is as following:
`DPAtomicModel`(`BaseModel`, `BaseAtomicModel`(torch.Tensor))
`PairTabModel`(`BaseModel`, `BaseAtomicModel`(torch.Tensor))

Now only `DPAtomicModel` has implemented the `compute_or_load_stat` method with descriptors and fittings.

I think this way may allow users implement different stat methods for model such as `PairTabModel` and not implement the numpy stat interface.

But now in this PR may be not suitable directly as an abstract method in `BaseModel`? (Or I can make it an abstract method and raise `NotImplement` in `PairTabModel` waiting for @anyangml fix?).